sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -541,12 +541,55 @@ def test_binomial_rewrite():
     k = Symbol('k', integer=True)
     x = Symbol('x')
 
-    assert binomial(n, k).rewrite(
```

Comment:
please leave the lines wrapped to less than 73 chars

Reply:
strictly 73 ?